localstack-core/localstack/utils/archives.py

```diff
@@ -1,21 +1,14 @@
-import io
-import tarfile
-import zipfile
-from subprocess import Popen
-from typing import IO, Optional
-
-try:
-    from typing import Literal
```

Comment:
`Literal` was introduced in Python 3.9. Considering we don't support 3.8 anymore, I don't _think_ we need the try/except anymore.